app/frontend/components/app.tsx

```diff
@@ -29,6 +29,14 @@ const App = connect((state) => ({
   if (currentTab === 'exchange') {
     return <Exchange />
   }
+  if (currentTab === 'nufi') {
+    return (
+      <div className="wrap nufi-background">
+        <NufiPreviewPage />
+        <Footer />
```

Comment:
After discussion with Michal, the `<Navbar/>` should not be apperieng on the website